source/Halibut/HalibutRuntime.cs

```diff
@@ -317,6 +317,26 @@ public async Task DisconnectAsync(ServiceEndPoint endpoint, CancellationToken ca
             await connectionManager.DisconnectAsync(endpoint, log, cancellationToken);
         }
 
+        [Obsolete("Use DisposeAsync")]
+        public void Dispose()
```

Comment:
I wonder if it's worth putting a comment as to why we are doing this? In a year, I'd be likely to want to remote this again 😁

Reply:
@sburmanoctopus It wasn't even a year, the added comments became helpful within 105 days! https://octopusdeploy.slack.com/archives/C04GAH78491/p1710722048771119